tests-node/unit/broccoli/translation-reducer-test.js

```diff
@@ -7,6 +7,14 @@ const subject = require('../../../lib/broccoli/translation-reducer');
 
 describe('translation-reducer', function() {
   beforeEach(function() {
+    this.icuFixture = {
+      de: {
+        foo: 'Hallo {who}',
```

Comment:
Can you write a test that uses parameters.

`You have {numPhotos, plural, =0 {no photos.} =1 {one photo.} other {# photos.}}`

I'd expect it to not fail if the parameters are different, only should match on first param (key).